### DISCUSSION TITLE
隐语开源Meetup一周年专场7月22日上海见！

### DISCUSSION THREAD

**Discussion by halosecret00** [secretflow/secretflow#706] [General]
![图片](https://mmbiz.qpic.cn/sz_mmbiz_png/rczZsXZbI361hgcqghpXvHoG4RGnbau5M4gS1sL3DrfpuKOACWAJOQunDibmHSmbjOicKFqhqnwlDfOgVbMgQ2IQ/640?wx_fmt=png)

小伙伴们，📢「隐语开源一周年 Meetup 」即将来袭！🎉在一周年 Meetup 上，不仅会对隐语 1.0 版本进行详解，还有新鲜出炉的隐语 MVP 部署体验包，让你秒变高手！更有机会与隐私计算行业的大佬们面对面交流，探讨技术前沿、畅想未来。🚀一起来看看剧透吧～  

本次一周年 Meetup ，由隐语开源社区发起，在上海市科学技术协会的指导下，联合上海金融科技产业联盟、上海开源信息技术协会共同主办，获得了中国信通院云计算与大数据研究所、隐私计算联盟的广泛支持。现特邀广大关注数据要素和隐私计算的兴趣人士、用户、开发者 7 月 22 日在上海相聚。**详细的议程及参会报名见下图哟~**  

  

  

**本场内容亮点**

- 隐语开源一周年的契机也迎来了隐语 1.0 版本发布。本次开源一周年特别场 Meetup，也将与大家分享基于隐语 1.0 发布后，隐语社区的升级与未来规划。  
- 基于隐语 1.0 共探发展新思路：隐语 1.0 版本开启了隐语开源的发展新蓝图，现场将分享隐语 1.0 版本之后对于隐私计算应用任务调度、跨平台互联互通的思考，并基于隐语 1.0 版本后的开源蓝图，与大家一起共探隐语社区发展新思路，推动技术互联互通与易用性共建。  
- 隐语开源社区组织架构升级：隐语社区期望可以有更多的组织与开发者一起共建与深入研究隐私计算技术，在此基础上成立了隐语 SIG（Special Interest Group，特别兴趣小组），目前包括：隐私安全算法 SIG、TEE SIG、技术标准 SIG、硬件加速 SIG 四大 SIG 小组，并将在现场发布四大 SIG 小组与升级后的隐语开源社区组织架构，希望广大用户与开发者们可以隐语开源社区里与隐语一起共同成长，更好地实现价值。  
- 隐语「OSCP」二期开启：隐语社区一直非常欢迎也非常感谢社区小伙伴的贡献。在隐语 OSCP（Open Source Contribution Plan）一期活动中，社区小伙伴们参与到 SecretFlow、SPU、SCQL 等项目的代码开发、文档验证、文档翻译等任务中，非常感谢大家的贡献。目前「OSCP」一期正在火热进行中🔥。基于社区的热情与隐语 1.0 版本后的开源蓝图，剧透一下，在本次活动现场，隐语社区也将进行「OSCP」一期贡献者荣誉展示并发布「OSCP」二期任务。“众人拾柴火焰高”，相信在隐语社区所有小伙伴的共同努力下，隐语开源社区将更加生机勃勃。  

**参会报名**

活动名额有限，**快长按识别下图二维码**报名吧！活动报名审核通过后，将会以邮件的形式通知审核结果，请务必填写正确的邮箱信息并及时关注邮件内容哦！欢迎“拖家带口”，带上你的小伙伴一起来玩转隐私计算新技术～🚀🔥

![图片](https://mmbiz.qpic.cn/sz_mmbiz_png/rczZsXZbI35EpuUOzBibtNkUFCwmZJnyVaibmLeib56deMY0sFb5IKIjddG54JaYjFficHIKywMUp03LDAkxUnN5Lw/640?wx_fmt=png)

同时，此次一周年 Meetup 也将在「隐语SecretFlow」B 站账号进行同步直播，无法到场的小伙伴们记得预约直播哟～  

![图片](https://mmbiz.qpic.cn/sz_mmbiz_jpg/rczZsXZbI361hgcqghpXvHoG4RGnbau5zOAfYFs81zse5uic3Bib5142GLicFHgGGzajmqziazF4WDDiadGyQf8zXUg/640?wx_fmt=jpeg)

亲身参与更能真正感受开源的魅力和影响力。相信每一个来参加一周年 Meetup 的小伙伴都将收获满满，不仅能结交更多志同道合的朋友，还能收获更多技术、经验和灵感！赶快点击“**阅读原文**”，预约现场席位吧，我们「隐语开源一周年 Meetup」上见！

隐语官网：

https://www.secretflow.org.cn

隐语社区：

https://github.com/secretflow

https://gitee.com/secretflow

联系我们：

公众号：隐语的小剧场

B站：隐语secretflow

邮箱：secretflow-contact@service.alipay.com

**Comment by Heffie199** [comment]
我参与了呀，收获满满，认识了社区的产品经理
